app/views/shared/sections/_newsletter_signup.haml

```diff
@@ -0,0 +1,6 @@
+%section
+  .container
+    .row.mt-5.mb-5.text-center
+      .col
+        %div{:style => "background-color:red;"}
```

Comment:
I just have the background red to dev, would remove once that padding issue is resolved.